docs/tutorials/qsimcirq.ipynb

```diff
@@ -383,7 +383,7 @@
     "id": "OUjhKoRVZGKZ"
    },
    "source": [
-    "qsim performance can be tuned further by passing options to the simulator constructor. These options use the same format as the qsim_base binary - a full description can be found in the qsim [usage doc](https://github.com/quantumlib/qsim/blob/master/docs/usage.md)."
+    "qsim performance can be tuned further by passing options to the simulator constructor. These options use the same format as the qsim_base binary - a full description can be found in the qsim [usage doc](https://github.com/quantumlib/qsim/blob/master/docs/usage.md). The example below demonstrates enabling multithreading in qsim; for best performance, use the same number of threads as the number of CPUs on your machine."
```

Comment:
I think it should be the number of cores (or even the number of virtual (logical) cores).

Reply:
Done.